cmd/otelcol/Dockerfile

```diff
@@ -45,14 +53,22 @@ FROM scratch
 ENV SPLUNK_BUNDLE_DIR=/usr/lib/splunk-otel-collector/agent-bundle
 ENV SPLUNK_COLLECTD_DIR=${SPLUNK_BUNDLE_DIR}/run/collectd
 ENV PATH=${PATH}:${SPLUNK_BUNDLE_DIR}/bin
+ENV PATH=${PATH}:${SPLUNK_BUNDLE_DIR}/jre/bin
+# avoid otel-java/autoconfigure default env vars for jmx receiver
+ENV OTEL_TRACES_EXPORTER=none
+ENV OTEL_LOGS_EXPORTER=none
```

Comment:
Do we need to set these env vars for the deb/rpm packages?

Reply:
I think this should actually be set by the gatherer itself to avoid internal metrics (that aren't categorized) instead of adopting everywhere. Will update test to do this and open PR in that project.